app/src/utils/player.js

```diff
@@ -34,6 +34,8 @@ export function getPlayerBuild(build) {
       return 'F2P';
     case '10hp':
       return '10 Hitpoints Pure';
+    case 'zerker':
+      return 'Berserker';
```

Comment:
Code looks good, just wondering if "Berserker" is really the way we should call them, rather than "Zerker" 🤔

Reply:
Changed